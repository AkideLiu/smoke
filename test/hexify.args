fixtures/hexify
fixtures/hexify.rb
