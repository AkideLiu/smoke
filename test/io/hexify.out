character
  succeeded
word
  succeeded
sentence
  succeeded
paragraph
  succeeded

4 tests, 0 failures
